packages/module/patternfly-docs/content/extensions/component-groups/about-component-groups.md

```diff
@@ -0,0 +1,9 @@
+---
+section: extensions
+subsection: Component groups
+id: About component groups
+--- 
+
+The **patternfly/react-component-groups** extension contains a range of React components that are more complex than basic PatternFly components, often combining and adjusting multiple base components to address common use cases. 
+
+This extension replaces **RedHatInsights/frontend-components**. To migrate your product to this new implementation, you can refer to migration guide details in [the README in the React component groups repository](https://github.com/patternfly/react-component-groups#readme).  
```

Comment:
```suggestion
This extension replaces **RedHatInsights/frontend-components**. To migrate your product to this new implementation, you can refer to migration guide details in [the README in the React component groups repository](https://github.com/patternfly/react-component-groups/migration.md).  
```

Sounds good - like this? add the md file in a followup issue?

Reply:
yes that's perfect.